apps/admin-gui/src/app/shared/components/dialogs/create-sponsored-member-dialog/create-sponsored-member-dialog.component.ts

```diff
@@ -81,7 +90,28 @@ export class CreateSponsoredMemberDialogComponent implements OnInit {
       if (this.namespaceOptions.length === 0) {
         this.functionalityNotSupported = true;
       }
-      this.loading = false;
+
+      const availableRoles = ['SPONSOR'];
+      const complementaryObject = {
+        id: this.data.voId,
+        beanName: 'Vo'
+      }
+      const availableRolesPrivileges = new Map<string, any>();
+
+      this.guiAuthResolver.getRolesAuthorization(availableRoles, complementaryObject, availableRolesPrivileges);
+      this.findSponsorsAuth = availableRolesPrivileges.get(availableRoles[0]).readAuth;
+
+      if (this.findSponsorsAuth) {
```

Comment:
if this.findSponsorsAuth is false then dont show at all the select field for making somebody else sponsor